### ISSUE TITLE
List of admin1 units to be updated for Morocco please 

### ISSUE THREAD

**Issue by csaussier** [HopkinsIDD/covid19-interventions#101]
The current list of admin1 units available in the database (16 regions) is not up to date. The regions distribution was changed in 2015 by a decree (see appended).

Here is the updated list as per the 2015 decree: 
Tanger-Tétouan-Al Hoceïma
L’Oriental
Fès-Meknès
Rabat-Salé-Kénitra
Béni Mellal-Khénifra
Casablanca-Settat
Marrakech-Safi
Drâa-Tafilalet
Souss-Massa
Guelmim-Oued Noun 
Laâyoune-Sakia El Hamra 
Dakhla-Oued Ed-Dah

Thanks for uploading the updated list.
Christel

[Morocco_2015_décret fixant le nombre des régions.pdf](https://github.com/HopkinsIDD/covid19-interventions/files/4475585/Morocco_2015_decret.fixant.le.nombre.des.regions.pdf)

**Comment by scottyaz** [comment]
@csaussier - is this the full current list? all the former ones should be erased?

**Comment by csaussier** [comment]
Yes it's the full current list and the former ones should be erased.
